deploy/secrets-store-csi-driver-windows.yaml

```diff
@@ -0,0 +1,78 @@
+kind: DaemonSet
+apiVersion: apps/v1
+metadata:
+  name: csi-secrets-store
+spec:
+  selector:
+    matchLabels:
+      app: csi-secrets-store
+  template:
+    metadata:
+      labels:
+        app: csi-secrets-store
+    spec:
+      nodeSelector:
+        beta.kubernetes.io/os: windows
+      serviceAccountName: secrets-store-csi-driver
+      containers:
+        - name: node-driver-registrar
+          image: mcr.microsoft.com/oss/kubernetes-csi/csi-node-driver-registrar:v1.3.0-pipeline
```

Comment:
What is the equivalent of this image compare to `quay.io/k8scsi/csi-node-driver-registrar`? The latest at https://github.com/kubernetes-csi/node-driver-registrar/releases is `1.2.0`

Do we also need to update this tag for the linux deployment?

Reply:
> What is the equivalent of this image compare to quay.io/k8scsi/csi-node-driver-registrar? The latest at https://github.com/kubernetes-csi/node-driver-registrar/releases is 1.2.0

There is still no official build for windows yet. We need the build from master as that contains all the changes required to make it work on windows. `v1.3.0-pipeline` is building the master.

> Do we also need to update this tag for the linux deployment?

Yes, we should update to the latest available release. I'll open a PR to bump it up.